bin/release_merged_branch

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env ruby
+
+require "bundler/inline"
+gemfile do
+  source "https://rubygems.org"
+  gem "multi_repo", require: "multi_repo/cli"
+end
+
+require "fileutils"
+
+opts = Optimist.options do
+  opt :branch,        "The branch to merge into.",              :type => :string, :required => true
```

Comment:
Should this be `target_branch` instead?

Reply:
Possibly, but branch is consistent with the other release scripts such as for a new branch or new tag.  If we changed one I'd want to change all 3.